### ISSUE TITLE
Omnichromium: support the new code search

### ISSUE THREAD

**Issue by nfischer** [kalman/chrome-extensions#9]
Chromium codesearch is moving to https://source.chromium.org/chromium. Would it be possible to migrate omnichromium to use the new codesearch? It would also be great if you can support interactive search results for the new site, as it the extension currently does for http://cs.chromium.org/.

**Comment by kalman** [comment]
Hiya @nfischer it's been an age since I was on the Chrome team - would you be interested in taking over the extension?

**Comment by nfischer** [comment]
I don't think I'll have the bandwidth, unfortunately. I don't believe the new codesearch has a similar public API, so it might make sense for the extension to stay as-is for the time being. I just wanted to raise the issue here.

My workaround was to add `https://source.chromium.org/search?q=%s&sq=&ss=chromium` to Chrome's search engine list. This doesn't provide interactive suggestions, but it saves a redirect and it keeps `&originalUrl=https:%2F%2Fcs.chromium.org%2F` from being tacked onto the final URL.

**Comment by kalman** [comment]
Yeah. The lack of live suggestions is the reason I wrote it all in the first place of course (and it somewhat grew from there).

I hope that the new codesearch does have some kind of API, and for the record if anybody does want to take over the extension (for real) then by all means contact me and we can organise it.

**Comment by sangwoo108** [comment]
Hi, kalman. I really love your extension.  Recently, I noticed that the extension was delisted from the store and felt really sad.

I wonder if there was open APIs for the old CS and now we don't for new code search. if I'm going to revive this extension, I should scan network requests what the current CS sends, right?

**Comment by kalman** [comment]
Hi @sangwoo108 

> I wonder if there was open APIs for the old CS and now we don't for new code search. if I'm going to revive this extension, I should scan network requests what the current CS sends, right?

that is what I did originally, yes. and if there are no official APIs for code search (it seems like the closest is https://developers.google.com/code-search/reference) then you'll probably have to start there as well.

I thought that the extension had been delisted ages ago due to trademark infringement... from what I can tell from that email, it was the logo, which I just need to change I guess. there is now also the manifest v3 restrictions which adds more complexity.

i'm happy to do the un-fun work of a logo and manifest changes, but I don't think i'll have the time to debug code search.

**Comment by sangwoo108** [comment]
> i'm happy to do the un-fun work of a logo and manifest changes, but I don't think i'll have the time to debug code search.

Do you have any logo to replace?  If you're running out of time, maybe I can give you a hand in my free time, including manifest v3 and debugging stuffs.
